### PR TITLE
create fulfillOracleKeyHistoryData

### PR DESCRIPTION
**Pull request by grujawork**

### CODE REVIEW

File: contracts/ThalesRoyale/ThalesRoyale.sol [hunk 1]
```diff
@@ -507,6 +507,14 @@ contract ThalesRoyale is Initializable, ProxyOwned, PausableUpgradeable, ProxyRe
         emit NewNumberOfRounds(_rounds);
     }
 
+    function fulfillOracleKeyHistoryData(bytes32 _oracleKey) public onlyOwner {
```
Comment (danijelthales) [inline contracts/ThalesRoyale/ThalesRoyale.sol:510]: use external

File: contracts/ThalesRoyale/ThalesRoyale.sol [hunk 2]
```diff
@@ -507,6 +507,14 @@ contract ThalesRoyale is Initializable, ProxyOwned, PausableUpgradeable, ProxyRe
...
+        // season 1 -> season
+        for (uint i=1; i<=season; i++) {
```
Comment (danijelthales) [inline contracts/ThalesRoyale/ThalesRoyale.sol:512]: season should be a parameter

Comment (danijelthales) [reply contracts/ThalesRoyale/ThalesRoyale.sol:512]: We should make sure that we only set it if it was empty before

**Review by danijelthales — CHANGES_REQUESTED** [review]

File: contracts/ThalesRoyale/ThalesRoyale.sol (hunk 2 above)
Comment (grujawork) [reply contracts/ThalesRoyale/ThalesRoyale.sol]: done

**Review by danijelthales — APPROVED** [review]

**Pull request merged**